pkg/maps/loader.go

```diff
@@ -470,31 +473,21 @@ func (m *BpfMap) BulkUpdateMapEntry(keyvalue map[uintptr]uintptr) error {
 
 func (m *BpfMap) BulkRefreshMapEntries(newMapContents map[string]uintptr) error {
 
-	// 1. Construct i/p to bulkMap
-	keyvaluePtr := make(map[uintptr]uintptr)
-
-	for k, v := range newMapContents {
-		keyByte := []byte(k)
-		log.Infof("Converted string to bytearray %v", keyByte)
-		keyPtr := uintptr(unsafe.Pointer(&keyByte[0]))
-		keyvaluePtr[keyPtr] = v
-	}
-
-	// 2. Update all map entries
-	err := m.BulkUpdateMapEntry(keyvaluePtr)
+	// 1. Update all map entries
```

Comment:
thinking about it, value field is still `uintptr` ..Now, it might not fail during map update because key has valid data but value might have incorrect data and there is no way we will know unless we manually check

Reply:
yeah, I did thought about this and better to send value byte array..